src/topp/DatabaseSuitability.cpp

```diff
@@ -64,7 +65,7 @@ class DatabaseSuitability :
 {
 public:
   DatabaseSuitability() :
-    TOPPBase("DatabaseSuitability", "Computes a suitability score for a database which was used for a peptide identification search. Also reports the quality of LC-MS spectra.", false)
```

Comment:
You can actually specify the citation in the constructor, such that it is correctly formatted and printed when running it on the command line.

Reply:
(as a separate parameter to the constructor, I mean)